File: Pranai/Arrays/Contains_duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int,greater<int>> s;
        for(auto i:nums)
            s.insert(i);
        if(s.size()==nums.size())
            return false;
        else
            return true;
    }
};

File: Adarsh/ContainsDuplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Map<Integer, Integer> m = new HashMap<Integer, Integer>();
        
        for(int i = 0; i < nums.length; i++) {
            if(m.get(nums[i]) != null)
                return true;
            else {
                m.put(nums[i], i);
            }
        }
        
        return false;
    }
}